### PR TITLE
Implemented program summary, supplement pack, fixed many issues

### LINKED ISSUE #16: (UI) Convert Login page html to VueJs component
**Description**

1. Convert the login page html to VueJs component 
2. Have Login button along with NGO and Admin button as it is for now

### LINKED ISSUE #71: (UI) Child Supplementary summary page
**Acceptance criteria:**

1) User should be able to see all the current child supplementary details
2) On clicking of Childs name it should navigate to Child Manage page
3) On clicking of "+" against each child user should be navigated to child nutritional data capture page
4) On Clicking of "New" the user should be navigated to New child addition details page

### LINKED ISSUE #99: (UI) Add Supplement pack details
Reference mock:

Image

### LINKED ISSUE #103: (UI) Program summary page implementation

### PR DESCRIPTION
**Pull request by mjayakumarmca**
Implemented,

1) Program Summary page
2) Supplement supply page
3) Fixed program summary API endpoint
4) Fixed kids check issues

### CODE REVIEW

File: FrontEnd/src/router/index.js [hunk 1]
```diff
@@ -74,22 +89,22 @@ const router = createRouter({
     {
       path: "/editSupplement/:id",
       name: "editSupplementPage",
-      component: () => import("../views/EditSupplement.vue")
+      component: () => import("../views/EditSupplement.vue"),
```
Comment (JaswinderSinghdigwa) [inline FrontEnd/src/router/index.js:92]: coma is their at the end of line

File: FrontEnd/src/router/index.js [hunk 2]
```diff
@@ -74,22 +89,22 @@ const router = createRouter({
...
     },
     {
       path: "/addDonor",
       name: "addDonor",
-      component: () => import("../views/AddDonor.vue")
+      component: () => import("../views/AddDonor.vue"),
```
Comment (JaswinderSinghdigwa) [inline FrontEnd/src/router/index.js:97]: same here

File: FrontEnd/src/stores/SupplmentarySummary.js [hunk 3]
```diff
@@ -16,15 +16,77 @@ export const useSupplmentarySummaryStore = defineStore("SupplmentarySummary", {
...
-        console.log(childMalnutritions.data.data);
-        //Todo: WIP checking in now for making sure other integrations are working fine
+        //let loadedChildren = JSON.parse(childs.data.data);
+        let loadedChildren = childs.data.data;
+        let modifedChildren = [];
 
-        // let filtered = childMalnutritions.filter(malnutrition =>        // filter jsondata
-        // childs.every( child =>                // so every member of filter array
-        //     child.value.includes(malnutrition[child.id])) )
+        for (let i in loadedChildren) {
```
Comment (JaswinderSinghdigwa) [inline FrontEnd/src/stores/SupplmentarySummary.js:22]: fixed alignment and one line space

**Review by Sunanda123-png — APPROVED** [review]
LGTM backend

File: FrontEnd/src/stores/SupplmentarySummary.js [hunk 4]
```diff
@@ -16,15 +16,77 @@ export const useSupplmentarySummaryStore = defineStore("SupplmentarySummary", {
...
+          } else {
+            sex = "F";
+            parentName = "D/O " + loadedChildren[i].motherName;
+          }
+          loadedChildren[i].gender = sex;
+          loadedChildren[i].motherName = parentName;
 
-        // const filteredChildMalnutritions = childMalnutritions.filter(malnutru => )
-        // this.childSummaries = data.data;
+          let filteredChildMalnutritions = childMalnutritions.data.data.find(
```
Comment (JaswinderSinghdigwa) [inline FrontEnd/src/stores/SupplmentarySummary.js:35]: fixed the alignment

File: FrontEnd/src/views/DashboardView.vue [hunk 5]
```diff
@@ -6,43 +6,60 @@
...
+                  <h3 class="card-title pricing-card-title">
+                    7,812
+                    <small class="text-muted fw-light"
+                      >[MAM] <i class="bi bi-arrow-down-circle text-success"></i> 123
+                    </small>
+                  </h3>
+                  <h3 class="card-title pricing-card-title">
+                    2,306
+                    <small class="text-muted fw-light"
+                      >[SAM] <i class="bi bi-arrow-up-circle text-danger"></i> 37
```
Comment (JaswinderSinghdigwa) [inline FrontEnd/src/views/DashboardView.vue]: >-CLOSING TAG SHOULD BE IN LINE 33

Comment (mjayakumarmca) [reply FrontEnd/src/views/DashboardView.vue]: This is because the IDE automatically formats like that.

File: FrontEnd/src/router/index.js (hunk 1 above)
Comment (mjayakumarmca) [reply FrontEnd/src/router/index.js:92]: It is there for all the lines even before and that is the way we should give

File: FrontEnd/src/stores/SupplmentarySummary.js (hunk 4 above)
Comment (mjayakumarmca) [reply FrontEnd/src/stores/SupplmentarySummary.js:35]: this is automatically done by IDE visual studio code.

**Pull request merged**